### DISCUSSION TITLE
Not able to install "SEB_3.3.2.413_x64_Setup.msi" file once installed the "SEB_3.3.2.413_SetupBundle.exe" file.

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#495] [Q&A]
Hi Team,

I have installed "SEB_3.3.2.413_SetupBundle.exe" on my laptop. Here the screenshot I have attached below.

![Screenshot from 2022-10-28 14-45-35](https://user-images.githubusercontent.com/111885140/198552562-c58f26d6-d164-4b6a-91ed-7059267bc31a.png)

But, While I try to install the "SEB_3.3.2.413_x64_Setup.msi", it was not installed. It shows "Repair" or "Remove". Please check the below screenshot.

![Screenshot from 2022-10-28 14-45-56](https://user-images.githubusercontent.com/111885140/198552822-3945474f-18b1-4032-be4c-fb54a601205b.png)

Why I could not able to install the .msi file?

But, In reverse it was installed. I can install the "SEB_3.3.2.413_x64_Setup.msi" first and then installed "SEB_3.3.2.413_SetupBundle.exe" both applications have been installed.

So, If I open the SEB Config file, from which application the config file will open, Will the SEB Config file open through the SEB.exe file? or through the SEB.msi file?

Can you please clarify on this.

**Comment by dbuechel** [comment]
You don't need nor will be able to install both the setup (.msi) as well as the setup bundle (.exe). The setup bundle (.exe) contains both the 32-bit as well as the 64-bit build of SEB and automatically installs the correct one for the respective system (hence our remark that everyone should use the setup bundle in the release notes). Furthermore, the setup bundle (.exe) also installs all required runtime dependencies of SEB, which the setup (.msi) does not.

Regardless of which setup you use, the installed SEB will always be exactly the same, and a SEB configuration will always be started by SEB's main executable `SafeExamBrowser.exe`.
